clang/test/Driver/cxx23-modules-print-library-module-manifest-path.cpp

```diff
@@ -0,0 +1,40 @@
+// Test that -print-library-module-manifest-path finds the correct file.
```

Comment:
Is the `cxx23-` prefix useful?

Reply:
not per se, I thought this was common in clang, I'll remove it.